### ISSUE TITLE
use multiple mesh in analysises

### ISSUE THREAD

**Issue by FreeCAD-Bug-Importer** [FreeCAD/FreeCAD#6159]
Issue imported from https://tracker.freecad.org/view.php?id=4560

* **Reporter:** Jee-Bee
* **Date submitted:** 2/11/2021
* **FreeCAD version:** 0.2
* **Category:** Feature
* **Status:** acknowledged
* **Tags:** 

# Original report text

ATM it is not possible to solve analysises with multiple meshes

# Additional information

https://forum.freecadweb.org/viewtopic.php?p=476836

# Steps to reproduce

-

# Other bug information

* **Priority:** normal
* **Severity:** feature
* **Category:** Feature
* **Updated:** 2/15/2021

# Discussion from Mantis ticket

### Comment by berndhahnebach 2021-02-15 05:54
BTW: it is possible to solve tie constraints with gaps in the surface. All bodies have to be meshed in the same huge mesh. 

But still ... The use of multiple meshes would be an great enhancement.

**Comment by SYehChen** [comment]
I create an example model for test. Here is the model and a pdf file for descripting the details of the model.
There are 4 parts in the model, glue together by ConnstarintTied.
. FreeCAD FEM complaint that "multiple mesh is not supported".

[Shaft_FEM_test.zip](https://github.com/FreeCAD/FreeCAD/files/10029044/Shaft_FEM_test.zip)

**Comment by luzpaz** [comment]
@SYehChen can you bump the forum thread as well to let them know you've uploaded an example? TIA!

**Comment by SYehChen** [comment]
I already did.

**Comment by luzpaz** [comment]
@donovaly can you take a look if you have a moment?

**Comment by donovaly** [comment]
> @donovaly can you take a look if you have a moment?

Not the next weeks. Sorry.

**Comment by luzpaz** [comment]
@berndhahnebach can you take a look at this?

**Comment by maxwxyz** [comment]
@FEA-eng FYI, this regularly comes up

**Comment by FEA-eng** [comment]
> @FEA-eng FYI, this regularly comes up

Yes, I often have to explain it on the forum and I described the current workarounds here: https://wiki.freecad.org/FEM_Geometry_Preparation_and_Meshing#Assembly_geometries
